### PR TITLE
bench: score_v2 change-region metric + null-baseline skill score

### PR DESCRIPTION
**Pull request by arthrod**
PR 3/13 of the robustness stack (stacks on #4).

The v1 pixel metric spends most of its dynamic range on unchanged body text — the do-nothing baseline historically scored ~68 mean, above half the leaderboard. Two **informational** parallel metrics, without touching the parity-locked scorer (rankings stay on ITT/pagefair):

- `null_score` — what rendering the BASE unchanged earns against the oracle, cached content-addressed in `results/null_baseline.json` (parent-process single-writer merge; pool workers never touch the file).
- `skill_score` — `(overall − null) / (100 − null) × 100`, clamped to [−100, 100]. 0 = no better than doing nothing; negative = worse. `None` on degenerate nulls (base ≈ oracle).
- `score_v2` — ink-F1 (2px tolerance) computed **only inside the change-region mask** (where base and oracle renders differ, dilated), ink-weighted across pages. On this chained-fixture corpus the mask often covers most of the page (spec review measured real nulls of 41–54), so v2 tracks v1 — its value concentrates on small-edit pairs and the upcoming mutation probes.

Per-line aggregates `skill_mean/median`, `v2_mean/median` (None-safe); RESULTS.md gains `skill_median`. Wiring is convention-based (mapping CSVs + `pdf_source` beside the oracle dir) — no new config keys; synthetic test corpora simply get `None`s. Base paths resolve in the parent before pool fanout (no callable pickling).

Tests: 10 new (`tests/test_score_v2.py`). Full suite: 246 passed.

🤖 Generated with [Claude Code](https://claude.com/claude-code)

## Summary by Sourcery

Introduce change-region scoring and null-baseline skill metrics alongside existing pagefair scores, wiring them through the pipeline, aggregation, CLI, and markdown export while keeping core rankings unchanged.

New Features:
- Add null_score, skill_score, and score_v2 fields to per-document score results and compute them during PDF scoring.
- Introduce score_v2 module implementing change-region ink-F1 scoring, null-baseline skill score, base-PDF resolution, and a content-addressed null baseline cache.
- Resolve base PDFs by corpus convention in the CLI and pass them, along with cached null baselines, into the scoring pipeline.
- Surface aggregated skill_mean/median and v2_mean/median in the results schema and export skill_median into RESULTS.md tables.

Tests:
- Add a dedicated test suite for score_v2 and the new skill/null/v2 metrics, including PDF fixtures, cache behavior, pipeline integration, and aggregate statistics.

### CODE REVIEW

**Comment by arthrod** [comment]
Closing: tip was already an ancestor of origin/main before #17 (landed earlier via the pure-juice merge path / intermediate stack merges). No unique commits remain to merge.

File: tests/test_score_v2.py [hunk 1]
```diff
@@ -0,0 +1,192 @@
...
+
+
+def test_do_nothing_candidate_scores_near_zero_on_v2_and_skill(trio, tmp_path):
+    result = pipeline.score_pdf_pair(
+        trio["oracle"], trio["do_nothing"], tmp_path / "w2", base_pdf=trio["base"],
+    )
+    # The candidate IS the null baseline: skill ~ 0, and the change region contains
+    # none of the inserted clause's ink: v2 ~ 0.
+    assert result["skill_score"] == pytest.approx(0.0, abs=5.0)
+    assert result["score_v2"] < 20.0
```
[the remark this answers is not in the document]
Comment (arthrod) [reply tests/test_score_v2.py:77]: Fixed in 5e1620d. Asserts `null_score ≈ overall_score` for the do-nothing candidate.
